cmd/ci/main.go

```diff
@@ -0,0 +1,354 @@
+package main
+
+import (
+	"encoding/json"
+	"os"
+	"path/filepath"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/urfave/cli/v2"
+)
+
+type TestGroup struct {
+	Name   string   `json:"name"`
+	Runner []string `json:"runner"`
```

Comment:
`runners` instead of `runner` since there may be multiple?

Reply:
It's still one. I'll make `[]string` into a `Runner` type so that it's clearer.